shell/common/api/electron_api_url_loader.cc

```diff
@@ -462,6 +469,9 @@ void SimpleURLLoaderWrapper::Cancel() {
 }
 scoped_refptr<network::SharedURLLoaderFactory>
 SimpleURLLoaderWrapper::GetURLLoaderFactoryForURL(const GURL& url) {
+  if (electron::IsUtilityProcess()) {
+    return URLLoaderBundle::GetInstance()->GetSharedURLLoaderFactory();
+  }
```

Comment:
Since `browser_context_` can be null now, I think we should add an assertion before using `browser_context_`, so if anything goes wrong we get an assertion instead of wild pointer crash.

```suggestion
  }
  CHECK(browser_context_);
```